src/baldr/graphtile.cc

```diff
@@ -111,9 +111,6 @@ GraphTile::GraphTile(const std::string& tile_dir, const GraphId& graphid) : head
         graphtile_.reset();
         return;
       }
-      std::ofstream foo(file_location + ".foobar", std::ios::binary | std::ios::trunc);
```

Comment:
this was a huge bug that i slipped in when testing my changes for zlib/gzip support not via boost iostreams